app/views/shows/index.html.haml

```diff
@@ -0,0 +1,5 @@
+- @shows.each do |show|
+	%h1= show["title"]
+	%h3= show["type"]
+	%end 
+	
```

Comment:
Your newline might not have worked here if it has indentation on it. Github still thinks there's no empty line at the end of the file